.github/dependabot.yml

```diff
@@ -9,3 +9,8 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "npm"
```

Comment:
This will automatically update the declared versions, but it may be worth having another pipeline on dependabot branches (related to npm?) that will rebuild and package the assets using the updated versions - probably a separate ticket.

Reply:
This this would be a good idea, but I think a one to revisit later or as a separate ticket. This is because the plan going forward has been decided to move to use webjars instead of NPM once the MoJ frontend packages with compiled stylesheets.